Dockerfile

```diff
@@ -18,9 +18,9 @@ COPY main.go main.go
 COPY controllers/ controllers/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
```

Comment:
Is this change related? If we need to revert the image change, it will also revert this change. If you split it to 2 commits we don't have this issue.

Reply:
Done